packages/server/README.md

```diff
@@ -27,11 +32,11 @@ The following tags are available:
 - Specific minor versions, such as `2.1`, `2.2`, `2.3-beta`
 - Specific patch versions, such as `2.0.4`, `2.2.8-beta`
 
-## Setup the database
+## Setup the Database
```

Comment:
Please leave the titles lowercase, that's just a convention we have in this project.